harness/determined/_experiment_config.py

```diff
@@ -26,6 +26,22 @@ def slots_per_trial(self) -> int:
     def experiment_seed(self) -> int:
         return int(self.get("reproducibility", {}).get("experiment_seed", 0))
 
+    def profiling_enabled(self) -> bool:
+        return bool(self.get("profiling", {}).get("enabled", False))
+
+    def profiling_interval(self) -> Tuple[int, Optional[int]]:
+        end_after_batch = self.get("profiling", {}).get("end_after_batch", None)
```

Comment:
I don't understand why you are keying off of `end_after_batch` rather than `enabled`?  And isn't `end_after_batch = 0` a totally valid value for `end_after_batch`?

Reply:
fixed